### PR TITLE
Issue #SB-0000 fix: All contents are not fetched and displayed in workspace

### PR DESCRIPTION
**Pull request by devendra-tarento**
# SunbirdEd - Portal
---

### Question Sets with status (Draft, Review, Published) are not appearing in Draft, Submitted For Review, Up For Review and Published section under Workspace for Creator and Reviewer..
---

#### Example
- [x] Applicable
- [ ] Not applicable

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Enhancement (additive changes to improve performance)
- [ ] This change requires a documentation update

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/app/client/src/app/modules/shared/services/config/app.config.json [hunk 1]
```diff
@@ -924,6 +924,7 @@
       "application/vnd.ekstep.content-collection"
     ],
     "objectType": "Content",
+    "allObjectType": ["Content", "QuestionSet"],
```
Comment (rajeevsathish) [inline src/app/client/src/app/modules/shared/services/config/app.config.json:927]: please make sure this name is more proper like allowedObjectTypes

**Review by rajeevsathish — CHANGES_REQUESTED** [review]

File: src/app/client/src/app/modules/workspace/components/draft/draft.component.ts [hunk 2]
```diff
@@ -229,15 +237,19 @@ export class DraftComponent extends WorkSpace implements OnInit, AfterViewInit {
             query: _.toString(bothParams['queryParams'].query),
             sort_by: this.sort
         };
+        if(!this.isQuestionSetEnabled) {
+            searchParams.filters["mimeType"] = this.config.appConfig.WORKSPACE.mimeType;
```
Comment (rajeevsathish) [inline src/app/client/src/app/modules/workspace/components/draft/draft.component.ts:241]: what will be the default value of the mimeType

**Review by rajeevsathish — CHANGES_REQUESTED** [review]

File: src/app/client/src/app/modules/shared/services/config/app.config.json (hunk 1 above)
Comment (devendra-tarento) [reply src/app/client/src/app/modules/shared/services/config/app.config.json]: Updated the name.

File: src/app/client/src/app/modules/workspace/components/draft/draft.component.ts [hunk 3]
```diff
@@ -177,6 +181,11 @@ export class DraftComponent extends WorkSpace implements OnInit, AfterViewInit {
         };
     }
     ngOnInit() {
+        this.workSpaceService.questionSetEnabled$.subscribe(
+            (response: any) => {
+              this.isQuestionSetEnabled = response.questionSetEnablement;
```
[the remark this answers is not in the document]
Comment (devendra-tarento) [reply src/app/client/src/app/modules/workspace/components/draft/draft.component.ts]: Added ? operaor.

**Pull request merged**